pkg/s3-proxy/config/manager.go

```diff
@@ -7,7 +7,7 @@ import "github.com/oxyno-zeta/s3-proxy/pkg/s3-proxy/log"
 //go:generate mockgen -destination=./mocks/mock_Manager.go -package=mocks github.com/oxyno-zeta/s3-proxy/pkg/s3-proxy/config Manager
```

Comment:
As you changed the interface, test mocks needs to be rebuilt. Can you start the generation please ?

Reply:
I did not take into account the fact that I would break some tests, I'll make sure to make them work again with my version.